src/Microsoft.AspNet.Owin/OwinExtensions.cs

```diff
@@ -67,21 +68,36 @@ public static IApplicationBuilder UseOwin(this IApplicationBuilder builder, Acti
 
         public static IApplicationBuilder UseBuilder(this AddMiddleware app)
         {
+            return app.UseBuilder(configureServiceProvider: null);
+        }
+
+        public static IApplicationBuilder UseBuilder(this AddMiddleware app, Action<IServiceCollection> configureServices)
```

Comment:
Might be best to just kill this overload and remove the dependency altogether, then...